docs/src/_tutorials/jupyter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: How to use jupyter with Meltano
+description: Using jupyter Python notebook as transformation tooling or to run analyses across your meltano maintained data.
+layout: doc
+weight: 4
+---
+
+As of _Sep-2022_, jupyter is not yet available as utility on the [hub](https://hub.meltano.com/), so you need to add it to your Meltano project as (local) custom plugin. Even if it is available as plugin on the hub, you will probably want to do some customization, so it will make sense to create your own version.
```

Comment:
```suggestion
As of _Sep-2022_, jupyter is not yet available as utility on the [Meltano Hub](https://hub.meltano.com/), so you need to add it to your Meltano project as (local) custom plugin. Even if it is available as plugin on the Hub, you will probably want to do some customization, so it will make sense to create your own version.
```